### PR TITLE
Git Issue 1948: LoadLevel Console Command Requires Full Asset Path Instead of Level Name

### PR DESCRIPTION
**Pull request by AMZN-Gene**
Update LoadLevel to also search inside the Level folder for a spawnable if LevelName isnt a fullpath to a spawnable.

Example: 
LoadLevel MyLevel
   _instead of..._
LoadLevel Levels/MyLevel/MyLevel.spawnable

fixes https://github.com/o3de/o3de/issues/1948
Signed-off-by: Gene Walters <genewalt@amazon.com>

### CODE REVIEW

File: Code/Legacy/CrySystem/LevelSystem/SpawnableLevelSystem.cpp [hunk 1]
```diff
@@ -174,19 +174,55 @@ namespace LegacyLevelSystem
...
+
+        if (rootSpawnableAssetId.IsValid())
+        {
+            validLevelName = levelName;
+        }
+        else
+        {
+            // It's common for users to only provide the level name, but not the full asset path
+            // Example: "MyLevel" instead of "Levels/MyLevel/MyLevel.spawnable"
+            if (strstr(levelName, ".spawnable") == nullptr)
```
Comment (lumberyard-employee-dm) [inline Code/Legacy/CrySystem/LevelSystem/SpawnableLevelSystem.cpp:191]: This should use AZ::IO::PathView for determing the extension `AZ::IO::PathView(levelName).Extension() == ".spawnable"`

**Review by lumberyard-employee-dm — COMMENTED** [review]
Is the the development repo the intended branch?

**Comment by AMZN-Gene** [comment]
> Is the the development repo the intended branch?

Oops, should this go into main? It's not critical, so this shouldn't go into stabilization.

**Comment by lumberyard-employee-dm** [comment]
I am just making sure your intent was to merge to the development branch.
It sounds like it is, so this is fine.

File: Code/Legacy/CrySystem/LevelSystem/SpawnableLevelSystem.cpp (hunk 1 above)
Comment (AMZN-Gene) [reply Code/Legacy/CrySystem/LevelSystem/SpawnableLevelSystem.cpp]: fixed!

**Review by lawsonamzn — APPROVED** [review]

**Review by lumberyard-employee-dm — APPROVED** [review]

**Pull request merged**